website/docs/components/text/partials/code/how-to-use.md

```diff
@@ -0,0 +1,137 @@
+The `Text` component is a **code-only** component used to apply specific predefined HDS typographic styles to a block of text or content.
+
+The equivalent if Figma is not a component, but a set of [typographic Figma styles](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=1262-9192).
+
+## How to use this component
+
+The “base” `Text` component is specialized in three different variants–the ones actually used by the consumers–with a one-to-one association between the namespace (eg. `::Display`) and the [corresponding style in Figma](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=1262-9192&mode=design).
+
+```handlebars
+<Hds::Text::Display>Text as "Display" variant</Hds::Text::Display>
+<Hds::Text::Body>Text as "Body" variant</Hds::Text::Body>
+<Hds::Text::Code>Text as "Code" variant</Hds::Text::Code>
+```
+
+Since no `@tag` argument is provided (see below) by default the text element is rendered as a `<span>`.
```

Comment:
Sometimes this directional language can be problematic (say, if the content is reorganized for some reason). Might be better to remove this or just link to the headline.

```suggestion
Since no `@tag` argument is provided by default the text element is rendered as a `<span>`.
```